tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1917,9 +1918,14 @@ def isDirectToVgprDoable(state, tc):
       reject(state, "DirectToVgpr%c does not supports InnerUnroll>1"%(tc))
       return False
 
+    # many DTVA failed generating kernels in these cases.
+    if tc == 'A' and (state["ProblemType"]["TLUA"] == state["UnrollMajorLDSA"]):
+      reject(state, "DirectToVgprA does not supports TLUA = UnrollMajorLDSA (different pattern of A)")
+      return False
+
```

Comment:
Lots of kernels failed to be generated or validated in such case: either failed in writer stage or validation stage. Can open another PR to fix them deeply. Simply reject these kinds of solutions.